astroquery/simbad/core.py

```diff
@@ -936,6 +1016,256 @@ def query_objectids_async(self, object_name, *, cache=True,
 
         return response
 
+    def list_tables(self, *, get_adql=False):
+        """The names and descriptions of the tables in SIMBAD.
+
+        Parameters
+        ----------
+        get_adql : bool, optional
+            Returns the ADQL string instead of querying SIMBAD.
+
+        Returns
+        -------
+        `~astropy.table.table.Table`
```

Comment:
interesting that this link works, I would expect a sphinx warning and therefore a red CI status for this
```suggestion
        `~astropy.table.Table`
```

Reply:
there are other occurances, those should be fixed, too.

Reply:
Done with https://github.com/astropy/astroquery/pull/2856/commits/d2b2bc4b8325e713a40d3150735d30d42e22760f :slightly_smiling_face: